### PR TITLE
slack integration item wrapper redesign

### PR DESCRIPTION
**Pull request by unicornzero**
## Description
Implement redesign of slack-integration-item:
* Styled the container that appears for each Slack channel we're connected to.
* Styled the container Reminder to appear when the channel integration has not been configured

## Pictures

image

## Reviewing this PR

_**Is Covered by Automated Tests?: YES**_

### CODE REVIEW

File: app/templates/components/forms/slack-integration-config.hbs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <div data-test-usage-notification-setting-form>
+  <p class='label-body'>Configure what Percy sends to {{channelName}} in your {{teamName}} Slack team.</p>
```
Comment (cadeParade) [inline app/templates/components/forms/slack-integration-config.hbs]: I am not 💯 on this, but it seems like `channelName` and `teamName` are not defined in this context?

File: app/templates/components/organizations/integrations/slack-integration-item.hbs [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
+    <div class="my-2">
+      {{#if hasConfigs}}
         {{#each slackIntegrationConfigs as |slackIntegrationConfig|}}
           {{organizations/integrations/slack-config-item
             slackIntegrationConfig=slackIntegrationConfig
             projectOptions=projectOptions
             deleteSlackIntegrationConfig=deleteSlackIntegrationConfig
           }}
         {{/each}}
+      {{else}}
```
Comment (cadeParade) [inline app/templates/components/organizations/integrations/slack-integration-item.hbs]: You can technically have an `each` and and an `else`
```
{{#each whatever do |foo|}}
  do a thing to items in the list
{{else}}
  if there are no items then do this
{{/each}}
```

https://guides.emberjs.com/release/templates/displaying-a-list-of-items/#toc_empty-lists

That being said, I do not oppose this implementation.

File: tests/integration/components/organizations/integrations/slack/slack-integration-item-test.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    it('renders the empty integration correctly', async function() {
+      expect(SlackIntegrationItem.isVisible).to.equal(true);
+      expect(SlackIntegrationItem.reminder.isVisible).to.equal(true);
+      expect(SlackIntegrationItem.addProjectButton.isVisible).to.equal(true);
+      expect(SlackIntegrationItem.deleteIntegrationButton.isVisible).to.equal(true);
+
+      await percySnapshot(this.test.fullTitle());
+    });
+
+    it('can delete the integration');
```
Comment (cadeParade) [inline tests/integration/components/organizations/integrations/slack/slack-integration-item-test.js]: You can use `it.skip` to skip the test.

File: app/templates/components/organizations/integrations/slack-integration-item.hbs (hunk 2 above)
Comment (unicornzero) [reply app/templates/components/organizations/integrations/slack-integration-item.hbs]: That's awesome syntax!

**Pull request merged**